### ISSUE TITLE
Windows - Path does not exist

### ISSUE THREAD

**Issue by Groumph** [layday/instawow#36]
### OS
Windows 10 2004

### Release
1.9

### Steps

1. Downloaded Windows 1.9.0 binary
1. Placed binary in its own folder
1. Ran the command `instawow reconcile --auto`
1. Configured add-on path and game flavour
1. Binary crashes after setting flavour

### Error
```
D:\instawow-classic> .\instawow.exe reconcile --auto
Traceback (most recent call last):
  File "instawow.__main__", line 4, in <module>
  File "click.core", line 829, in __call__
  File "click.core", line 782, in main
  File "click.core", line 1259, in invoke
  File "click.core", line 1066, in invoke
  File "click.core", line 610, in invoke
  File "click.decorators", line 21, in new_func
  File "instawow.cli", line 320, in reconcile
  File "instawow.utils", line 106, in __get__
  File "instawow.cli", line 121, in m
  File "instawow.manager", line 208, in prepare_db_session
  File "alembic.command", line 278, in upgrade
  File "alembic.script.base", line 149, in from_config
  File "alembic.script.base", line 70, in __init__
alembic.util.exc.CommandError: Path doesn't exist: 'C:\\Users\\<username>\\AppData\\Local\\Temp\\tmpdhbqt6ov\\instawow\\migrations'.  Please use the 'init' command to create a new scripts folder.
SystemError
```

### Notes
Same error exists every time I run reconcile.

**Comment by layday** [comment]
Thanks for the detailed report.  This appears to be an issue with the compiled version only for Windows but the stack trace does not offer much in the way of a clue as to what might be happening.  I'll fire up the ol' Windows VM and see what I can uncover.

**Comment by layday** [comment]
This will be fixed in 1.9.1 to be released shortly.  Thanks again for the report.
